Source/FortranInterface/WarpX_picsar.F90

```diff
@@ -301,6 +314,58 @@ subroutine warpx_current_deposition( &
 
   end subroutine warpx_current_deposition
 
+  ! _________________________________________________________________
+  !>
+  !> @brief
+  !> Applies the inverse volume scaling for RZ current deposition
+  !>
+  !> @details
+  !> The scaling is done for both single mode (FDTD) and
+  !> multi mode (spectral)
+  !
+  !> @param[inout] jx,jy,jz current arrays
+  !> @param[in] nmodes number of spectral modes
+  !> @param[in] rmin tile grid minimum radius
+  !> @param[in] dr radial space discretization steps
+  !> @param[in] nx,ny,nz number of cells
+  !> @param[in] nxguard,nyguard,nzguard number of guard cells
```

Comment:
`nx, ny, nz` and `nxguard, nyguard, nzguard` do not seem to be parameters for the function below. I would suggest to remove them.